src/algebra/bit manipulation.md

```diff
@@ -0,0 +1,214 @@
+---
+tags:
+  - Translated
+e_maxx_link: all_submask
+---
+
+# Bit operators
+
+When finding solutions to problems, a very important factor is the simplicity and the execution time. Now, in this bitwise operations are very important and you can take advantage of them.
+
+* ## Bit shift operators
+### Multiplication and division by powers of 2
```

Comment:
This section is very, very long for simply explaining that $x \ll y = x \cdot 2y$ and $x \gg y = x / 2^y$.
I would compress this a bit. 
And there is no need to make a new paragraph (via an empty newline in the middle) for every single sentence.